bin/partial_upload.py

```diff
@@ -0,0 +1,76 @@
+#!/bin/env python
+# -*- coding: utf8 -*-
+""" Triggers a partial upload process with the specified raw.xz URL. """
+
+import argparse
+import logging
+import logging.config
+
+import fedmsg.config
+
+from fedimg.config import AWS_ACCESS_ID
+from fedimg.config import AWS_SECRET_KEY
+from fedimg.config import AWS_BASE_REGION, AWS_REGIONS
+from fedimg.services.ec2.ec2copy import main as ec2copy
+from fedimg.services.ec2.ec2initiate import main as ec2main
+
+logging.config.dictConfig(fedmsg.config.load_config()['logging'])
+log = logging.getLogger('fedmsg')
```

Comment:
You have a logger but you are not using it in the script. I guess it could be nice to add some info and debug logs